std_msgs/Header Header

string robot_name
KeyFrame[] keyframes
Edge[] edges